### PR TITLE
Haskell Stack support

### PR DESCRIPTION
**Pull request by csabahruska**

### CODE REVIEW

File: sanitize_tabs_and_spaces.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+for ext in cabal hs yaml c h ll s; do
+  # removes trailing spaces
+  find . -name "*.$ext" -type f -print0 | xargs -0 sed -i 's/[[:space:]]*$//'
+
+  # expand tabs to spaces
+  find . -name "*.$ext" ! -type d -exec bash -c 'expand "$0" > /tmp/e && mv /tmp/e "$0"' {} \;
+done
```
Comment (robinvd) [inline sanitize_tabs_and_spaces.sh]: While definitly a usefull script, i don't think this should be in the repo.

**Review by robinvd — COMMENTED** [review]
Thanks for the pr! didnt think anyone would show interest in the project. I just have one comment and then i can be merged.

File: sanitize_tabs_and_spaces.sh (hunk 1 above)
Comment (csabahruska) [reply sanitize_tabs_and_spaces.sh]: OK. I deleted it.

**Comment by csabahruska** [comment]
This project is nice and small example for LLVM stack map based GC implementation.

**Comment by robinvd** [comment]
@csabahruska thanks for the compliment, hope you got some use out of it. Don't hesitate to comment/make an issue.

Thanks again for the PR

**Pull request merged**